Cql/CoreTests/ExpressionBuilderTests.cs

```diff
@@ -45,5 +46,26 @@ public void QueriesTest_1_0_0()
             var eb = new ExpressionBuilder(binding, typeManager, elmPackage, logger);
             var expressions = eb.Build();
         }
+
+        // https://github.com/FirelyTeam/firely-cql-sdk/issues/129
+        [TestMethod]
+        public void Medication_Request_Example_Test()
+        {
+            var binding = new CqlOperatorsBinding(TypeResolver, TypeConverter);
+            var typeManager = new TypeManager(TypeResolver);
+            var elm = new FileInfo(@"Input\ELM\Test\Medication_Request_Example.json");
+            var elmPackage = Hl7.Cql.Elm.Library.LoadFromJson(elm);
+            var logger = CreateLogger();
+            var eb = new ExpressionBuilder(binding, typeManager, elmPackage, logger);
+
+            var fdt = new FhirDateTime(2023, 12, 11, 9, 41, 30, TimeSpan.FromHours(-5));
+            var fdts = fdt.ToString();
+            var fs = new FhirDateTime(fdts);
+            Assert.AreEqual(fdt, fs);
+
+            
+            var expressions = eb.Build();

```

Comment:
your setting up an ExpressionBuilder, and then call Build(), but shouldn't you check something?

Reply:
Success for this test is that `Build()` doesn't throw any exceptions.

Reply:
I understand, but it's nice for the reader (and those that maintain the test for you, later), that that is explicit. I'll add a comment & null check for you.

Reply:
Weirdly enough, it did not build locally anymore because of localization and null-reference errors. Fixed that too. Maybe the new compiler that came with SDK 8 is a bit stricter than the one used in the CI build?  Anyway, fixed it.